gitcmd/pr.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/forbole/bdtool/types"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+func PullRequest(chain *types.ChainInfo, targetBranch string, GitHubToken string) error {
+	client := github.NewClient(oauth2.NewClient(
+		context.Background(),
+		oauth2.StaticTokenSource(
+			&oauth2.Token{
+				AccessToken: GitHubToken,
+			},
+		),
+	))
+
+	// Create PR
+	pr, _, err := client.PullRequests.Create(
+		context.Background(),
+		"forbole",
```

Comment:
```suggestion
		"forbole",
```
Also here, it would be better if the organisation name is read from the user input